ASAP/ASAP-paper.Rmd

```diff
@@ -24,7 +24,17 @@ lapply(pkgs, library, character.only = T)
 
 
 
-**Abstract** The analysis of large data sets, often referred to as big data, is increasingly being performed in research and the development of policy. Big data has the potential to provide new and innovative insights to existing problems, and to reveal trends and interactions that were not previously considered. This paper presents a big data analysis approach in an effort to determine the effectiveness of the Environmental Stewardship Scheme (ESS) agreements in place along the Pennine Way National Trail. We present a method to extract the sentiment conveyed in trail users' public Twitter messages (tweets) in order to assess to what extent ESS agreements maintain landscape character within the trail corridor. With this knowledge we provide recommendations regarding the feasibility of using social media big data as a source of trail user feedback that could be of utility to Natural England, the executive non-departmental public body of the UK Government charged with the management of both England's National Trail System and Environmental Stewardship Scheme.
+**Abstract**
+
+Large and unofficial data sets, often referred to as 'big data', are increasingly being used in geographical research and explored as a decision support tool for policy development.
+Big data has the potential to provide new insight into phenomena about which there is little information from conventional sources.
+Within this context, the present paper explores the potential of a large social media dataset to evaluate the perceptions of visitors to the Pennine Way Trail, which is protected by the UK's Environmental Stewardship Scheme (ESS).
+The method analyses sentiment in trail users' public Twitter messages (tweets) with the aim of assessing the extent to which the ESS maintains landscape character within the trail corridor.
+The method demonstrates the importance of filtering big data to convert the data into useful information.
+After filtering, the results are based on 96 messages directly related to the trail.
```

Comment:
Added the total number of messages up-front as it may be mentioned later on. Links with the importance of filtering, alluded to here: http://onlinelibrary.wiley.com/doi/10.1111/gean.12081/abstract